packages/components/src/FilterDatePicker/subcomponents/FilterDatePickerField/FilterDatePickerField.module.scss

```
@@ -0,0 +1,6 @@
+.FilterDatePickerField {
+  width: 288px;
+  display: flex;
+  flex-direction: column;
+  gap: theme("spacing.16");
```

??? Is this something we're doing? Usually we get the token and do this
```suggestion
  gap: $spacing-16;
```

---

I'm trying to go SCSS-less 😛 but after doing this I did think we should have design tokens generate a CSS version of tokens and use those instead.

---

I don't mind going scss-less, but probably worth getting everyone on board with the way to do it first :P